lib/Sema/CSSimplify.cpp

```diff
@@ -1208,6 +1208,17 @@ static bool allowsBridgingFromObjC(TypeChecker &tc, DeclContext *dc,
   return true;
 }
 
+/// Determine whether the given type variables occurs in the given type.
+static bool typeVarOccursInType(TypeVariableType *typeVar, Type type) {
```

Comment:
/me points out that now we have a better answer to @CodaFi 's question, "where's the occurs check"?